core/src/main/java/bisq/core/filter/Filter.java

```diff
@@ -91,6 +95,11 @@
     // added at v1.3.8
     private final boolean disableAutoConf;
 
+    // added at v1.5.5
+    // In contrast to bannedNodeAddress those addresses cannot access the network but are rejected immediately.
+    // Should be only used in case of dos attacks
+    private final Set<String> nodeAddressesBannedFromNetwork;
```

Comment:
Naming is confusing since there is a `bannedNodeAddress` and a `nodeAddressesBannedFromNetwork`, better to have more descriptive names.

Reply:
I think `nodeAddressesBannedFromNetwork` is pretty clear (and long ;-)) no? The `bannedNodeAddress` is not great but renaming it would require a rename if the protobuf field as well to avoid that those fields get out of sync and was not sure if thats worth it. Also not clear for a good alternative name.

Reply:
Renamed it to nodeAddressesBannedFromTrading

Reply:
Much better this way, no need for comments since code is self describing. Agreed that it was the `bannedNodeAddress` that needed a name change.